crates/gitbutler-branch-actions/src/branch.rs

```diff
@@ -22,82 +21,68 @@ pub fn list_branches(
     ctx: &CommandContext,
     filter: Option<BranchListingFilter>,
 ) -> Result<Vec<BranchListing>> {
+    let has_filter = filter.is_some();
+    let filter = filter.unwrap_or_default();
     let vb_handle = ctx.project().virtual_branches();
     // The definition of "own_branch" is based if the current user made the first commit on the branch
```

Comment:
On note here - the comment is already out of date unfortunately. I realized that the question that the UI is trying to answer is "what are the branches that the user is involved in" - and offer segmented control in the UI so that the user can more easily focus on their branches.
Thus the definition changed to "branches that the user has committed to" (and if they are virtual branches without commit that they created, also include that). The implementation below already reflects this definition, just not the comment here

Reply:
Alright, I will update the comment (or remove it, because I think that's tested now), and update the docs for the filter accordingly.